main.cpp

```diff
@@ -13,75 +13,104 @@ What you create in this project will be used as the basis of Project 5 in the co
-        1)
-        2)
-        3)
-        4)
-    Name 3 things you can do with the:   Instrument Cluster
-        1)
-        2)
-        3)
-    
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Environment Controls
-        1)
-        2)
-        3)
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Infotainment System
-        1)
-        2)
-        3)
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1)
-        2)
-        3)
-    Name 2 things you can do with the:   Seat
-        1)
-        2)
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - the amount of water used per week.
+        - amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+ */
+
+/*
+Thing 1) DAW
+5 properties:
+    1) stores digital data
+    2) prompts user input
+    3) hosts plugins
+    4) automates process
+    5) generates sound
+3 things it can do:
+    1) master music
+    2) process audio
+    3) mix music
+ */
+
+/*
+Thing 2) airport
+5 properties:
+    1) number of planes
+    2) employs floor satff
+    3) service aeroplanes
+    4) number of runways
+    5) tracks with radar
+3 things it can do:
+    1) land and takeoff planes
+    2) centralise travel departure
+    3) service travel industry
+ */
+
+/*
+Thing 3) farm
+5 properties:
+    1) number of chickens
+    2) number of cows
+    3) single farmer
+    4) the amount of feed per week
+    5) the amount of milk produced
+3 things it can do:
+    1) grow veg
+    2) rear animals
+    3) produce milk
+ */
+
+/*
+Thing 4) school
+5 properties:
+    1) number of students
+    2) number of teachers
+    3) hours per week of teaching
+    4) hours per week of lunch
+    5) separate classromms
```

Comment:
`separate classrooms` is not a property, it's a thing. 
pick something else, like `average number of blackboards per teacher'
- [ ] revise